README.md

```diff
@@ -326,6 +330,12 @@ spec:
         type: string # optional
         spec:
           # arbitrary chosen fields for every data source type to make it comfortable to use.
+    raw: # the precomputed ratio stored as a metric, can't be used together with good/bad/total
```

Comment:
I made this comment in the issue, but wouldn't this just be a `thresholdMetric`?

Reply:
`thresholdMetric` uses `objectives[].value` to compare all the data points of the SLI to determine the good ones. It does assume the metric itself is a ratio.